en/techdocs/api/client/hoodie.store.md

```diff
@@ -363,141 +415,184 @@ Returns undefined.
     showNewNoteAddedNotification();
   }
 });
+```
+
+You could also solve this with more specific listeners, one for each **type**:
+
+```javascript
 hoodie.store.on('todo:add', showNewTodoAddedNotification);
 hoodie.store.on('note:add', showNewNoteAddedNotification);
-</code></pre>
+```
 
 <a id="store"></a>
-### store()
+### [store()](#store)
 **version:**      *> 0.2.0*
 
-Creates a store instance that is scoped by type, or by type &amp; id.
+**Creates a so-called *scoped store* instance that is scoped by *type*, or by *type* and *id*.**
 
-<pre><code>hoodie.store('type', 'id');</code></pre>
+```javascript
+hoodie.store(type, id);
+```
 
 | option     | type   | description        | required |
-| ---------- |:------:|:------------------:|:--------:|
-| type       | String | type of the store  | yes      |
+|:---------- |:------ |:------------------ |:-------- |
+| type       | String | type of the object | yes      |
 | id         | String | index of store obj | no       |
 
-##### Example
+It is quite likely that your application will have more than one type of store object, and at some point you'll want to stop fiddling with specifying object types all the time. **Scoped stores** provide a handy shortcut to the **hoodie.store** methods, pre-scoped to a specific object **type**, or even a specific object (when you pass in an **id**).
+
+This means you can use **hoodie.store** methods directly on objects created with **hoodie.store()**.
+
+
+#### Example
 
-It is most likely, that your application will have more than one type of store objects. Even if you have just a single object hoodie.store(type) comes handy. Say you have to work with objects of the type todo, you usually do something like the following:
+Usually you'd do something like this in your todo app:
 
-<pre><code>hoodie.store.add('todo', { title: 'Getting Coffee' });
+```javascript
+hoodie.store.add('todo', { title: 'Getting Coffee' });
 hoodie.store.findAll('todo')
   .done(function(allTodos) { /* ... */ });
-</code></pre>
+```
```

Comment:
Oof, thanks for catching that. I should have made it clearer that `add` and `findAll` don't actually follow each other in the code, only in time. Your solution is a lot safer :)